internal/avs/external_eval_assistant.go

```diff
@@ -48,8 +48,63 @@ func (eea *ExternalEvalAssistant) ProvideParentId(_ internal.ProvisioningParamet
 	return eea.avsConfig.ParentId
 }
 
-func (eea *ExternalEvalAssistant) ProvideTags() []*Tag {
-	return eea.avsConfig.ExternalTesterTags
+// func (iec *InternalEvalAssistant) ProvideTags(pp internal.ProvisioningParameters, rt runtime.RuntimeDTO) []*Tag {
+func (eea *ExternalEvalAssistant) ProvideTags(operation internal.Operation) []*Tag {
+	var Tags []*Tag
+	Tags = append(Tags, &Tag{
+		Content:    operation.InstanceID,
+		TagClassId: eea.avsConfig.InstanceIdTagClassId,
+	})
+
+	Tags = append(Tags, &Tag{
+		Content:    operation.ProvisioningParameters.ErsContext.GlobalAccountID,
+		TagClassId: eea.avsConfig.GlobalAccountIdTagClassId,
+	})
+
+	Tags = append(Tags, &Tag{
+		Content:    operation.ProvisioningParameters.ErsContext.SubAccountID,
+		TagClassId: eea.avsConfig.SubAccountIdTagClassId,
+	})
+
+	Tags = append(Tags, &Tag{
+		Content:    operation.ProvisioningParameters.PlatformRegion,
+		TagClassId: eea.avsConfig.LandscapeTagClassId,
+	})
+
+	Tags = append(Tags, &Tag{
+		Content:    string(operation.ProvisioningParameters.PlatformProvider),
+		TagClassId: eea.avsConfig.ProviderTagClassId,
+	})
+
+	r := ""
+	if operation.ProvisioningParameters.ErsContext.Region != nil {
+		r = *operation.ProvisioningParameters.ErsContext.Region
```

Comment:
`operation.ProvisioningParameters.ErsContext.Region` doesn't hold the platform region value, you can remove it.

Reply:
ack!